dipy/align/tests/test_imaffine.py

```diff
@@ -312,7 +320,232 @@ def test_mi_gradient():
             expected[i] = (val1 - val0) / h
 
         dp = expected.dot(actual)
-        enorm = np.linalg.norm(expected)
-        anorm = np.linalg.norm(actual)
+        enorm = npl.norm(expected)
+        anorm = npl.norm(actual)
         nprod = dp / (enorm * anorm)
         assert(nprod >= 0.99)
+
+
+def create_affine_transforms(dim, translations, rotations, scales, rot_axis=None):
+    r""" Creates a list of affine transforms with all combinations of params
+
+    This function is intended to be used for testing only. It generates
```

Comment:
Yes - I would think something like that. Anyone else have strong opinions about this one?